packages/driver/cypress/integration/cy/snapshot_spec.js

```diff
@@ -166,4 +166,28 @@ describe('driver/src/cy/snapshots', () => {
       })
     })
   })
+
+  context('custom elements', () => {
+    beforeEach(() => {
+      cy.visit('/fixtures/custom-elements.html')
+    })
+
+    // https://github.com/cypress-io/cypress/issues/7187
+    it('does not trigger shadow dom constructor', () => {
```

Comment:
I hate to nitpick, but the test titles and the method names still reference shadow dom. I just don't want there to be any confusion for a future developer looking at these.

Reply:
Sorry, missed that. Definitely understand the need to clarify the difference, updated now.